CHANGELOG.rst

```diff
@@ -14,6 +14,11 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+
+[3.2.6] - 2020-11-05
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Remove soon-to-be-deprecated WaffleSwitchNamespace class instances
```

Comment:
What do this about the following?  Is it all accurate?
```
[4.0.0] - 2020-11-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
* BACKWARD INCOMPATIBLE: Removes ``waffle()``, which returned a (now deprecated) WaffleSwitchNamespace. This should only affect tests in edx-platform. 
* Requires edx-toggles>=2.0.0, which introduces backward-incompatible changes to WaffleSwitch.
* Refactors ``ENABLE_COMPLETION_TRACKING_SWITCH`` from a ``LegacyWaffleSwitch`` to the updated ``WaffleSwitch``.  We don't expect uses of this updated switch to require changes, unless there are
surprise uses of deprecated methods from ``LegacyWaffleSwitch``.
```

Reply:
Thanks, I'll use this.